### PR TITLE
Refreshed the useful L-band GEOs

### PR DESCRIPTION
**Pull request by adriaanph**
The following are deleted:
  ARTEMIS (retired)
  INMARSAT 3-F2 (retired)
  INMARSAT 3-F5 (relocated to 54degW - below our horizon)
  ALPHASAT (no suitable beacons, just a bright satellite)
The following are added:
EWS-G1 (GOES 13) (beacon at 1694MHz)

### CODE REVIEW

**Comment by adriaanph** [comment]
@lanceWilliams please check this - i can't assign reviewers and i need this for my RTS tests (currently have a work-around hack, so not urgent).

**Comment by adriaanph** [comment]
@samueltwum1  can you perhaps approve this for me?

**Review by samueltwum1 — APPROVED** [review]
LGTM

**Comment by adriaanph** [comment]
thanks samuel. do note that i don't have permission to merge, so i'm stuck, can you merge this PR?

**Comment by samueltwum1** [comment]
@adriaanph unfortunately I dont have write access either.

**Comment by adriaanph** [comment]
okay, thanks.

**Comment by samueltwum1** [comment]
@tockards perhaps you can help merge this PR if you have write access (maybe you can also grant @adriaanph  write access to this repo?). I crossed checked the satelite info updated and approved the PR.

**Comment by tockards** [comment]
@samueltwum1 we need someone who is a maintainer on the @ska-sa/cam group(see https://github.com/orgs/ska-sa/teams/cam/members?query=role%3Amaintainer) to give @adriaanph  permissions to merge these PRs. That request has to be sent to @ofaletse

**Pull request merged**
